### ISSUE TITLE
Uncaught TypeError: o is null with Step-by-step Highlights

### ISSUE THREAD

**Issue by Toshizuki** [Martinomagnifico/reveal.js-copycode#10]
Hi,

I have an Uncaught Type Error when using copycode with Step-step Highlights.

```
<pre><code data-line-numbers="3-5|8-10|13-15">
<table>
  <tr>
    <td>Apples</td>
    <td>$1</td>
    <td>7</td>
  </tr>
  <tr>
    <td>Oranges</td>
    <td>$2</td>
    <td>18</td>
  </tr>
  <tr>
    <td>Kiwi</td>
    <td>$3</td>
    <td>1</td>
  </tr>
</table>
</code></pre>
```
This code block generates the following error:
![Capture d’écran 2024-02-26 153909](https://github.com/Martinomagnifico/reveal.js-copycode/assets/161334313/e0963443-d9da-412b-afb1-53fbe163a262)

It is working well if I use `data-line-numbers="3-5,8-10,13-15"`.

Thank you for your time,

**Comment by Martinomagnifico** [comment]
Thanks for the catch, will look at the issue this weekend.

**Comment by Martinomagnifico** [comment]
Testing your markup, the above code will render as a table, because it isn't escaped. Did you try that? I also can't reproduce the issue. Send me a zip of your testproject so I can check it out.

**Comment by Toshizuki** [comment]
I am writing my slide in Markdown. Here is the slide that cause the issue:
```

---

## test copy

<pre><code data-line-numbers="3-5|8-10|13-15">
<table>
  <tr>
    <td>Apples</td>
    <td>$1</td>
    <td>7</td>
  </tr>
  <tr>
    <td>Oranges</td>
    <td>$2</td>
    <td>18</td>
  </tr>
  <tr>
    <td>Kiwi</td>
    <td>$3</td>
    <td>1</td>
  </tr>
</table>
</code></pre>
```
On my side, the rendering is good.
![image](https://github.com/Martinomagnifico/reveal.js-copycode/assets/161334313/7d55e361-038d-41ee-8bb8-bfa3ed7c2172)
But the copy doesn't work.
I have also tried the markdown version with:
```

---

## test with markdown

```js [1-2|3|4]
  let a = 1;
  let b = 2;
  let c = x => 1 + 2 + x;
  c(3);
\```
```
The \ is here only for the block code in github.
The rendering is also good:
![image](https://github.com/Martinomagnifico/reveal.js-copycode/assets/161334313/141a8289-48cd-455e-8663-619a46b27f52)
But I have the same issue.

I will send you an example project tomorrow.

Let me know if you need more information.
Thanks.

**Comment by Toshizuki** [comment]
Hello,
Here is an example project that reproduce my issue. 
[example.zip](https://github.com/Martinomagnifico/reveal.js-copycode/files/14432864/example.zip)
**How to reproduce:**

1. cd example && npm install
2. npm start
3. check on localhost:8000

I am able to reproduce the issue in my local environment or in gitlab pages.

**Comment by Martinomagnifico** [comment]
This is now fixed in version 1.1.8.

**Comment by Toshizuki** [comment]
Thank you for your work. This issue has been resolved.
